### DISCUSSION TITLE
Siren functionality in  HA?

### DISCUSSION THREAD

**Discussion by nickrout** [Koenkk/zigbee2mqtt#8284] [General]
Is the new siren functionality in home assistant able to be used/added to z2m functionality? I have a Tuya TS0216 which exposes battery, alarm, volume and lqi. There is no siren.* services in HA. 

The HA release notes indicate that siren has only been added for z-wave integration, so the lack of services doesn't surprise me, but will this be added?

**Comment by Morphy99** [comment]
I've got the same model and yes no siren service. I'm not sure the point in adding a such a service though it's just a switch with another name?

**Reply by Morphy99** [reply]
On another topic, does pressing your physical volume buttons trigger any kind of MQTT messages from the TS0216?
I also can't change the volume from HA but I don't think the siren service has this (yet?) either.

**Comment by Nardol** [comment]
MQTT siren is on road with Home Assistant 2022.3: https://rc.home-assistant.io/integrations/siren.mqtt/
